content/docs/details/entity-push-api/index.md

```diff
@@ -0,0 +1,205 @@
+
+response.then((r) => {
+  const accounts = r.Accounts;
+
+  if (!accounts) {
+    throw new Error('No AWS Accounts found');
+  }
+
+  const templateResourceEntity = ({ name, arn }) => ({
+    apiVersion: 'backstage.io/v1alpha1',
+    kind: 'Resource',
+    metadata: {
+      name,
+      description: 'AWS accounts',
+      annotations: {
+        'aws-account/arn': arn,
+      },
+    },
+    spec: {
+      owner: 'dx-team',
+      type: 'aws-account',
+    },
+  });
+
+  const entities = accounts.map((a) => templateResourceEntity({ name: a.Name, arn: a.Arn }));
+
+  for (entity of entities) {
+    const body = JSON.stringify(entity);
+    const req = https.request(
+      {
+        hostname: 'api.roadie.so',
+        port: 443,
+        path: '/api/catalog/roadie-entities/sets/aws-accounts',
+        method: 'PUT',
+        headers: {
+          'Content-Type': 'application/json',
+          'Content-Length': Buffer.byteLength(body),
+          Authorization: `bearer ${process.env.ROADIE_API_TOKEN}`,
+        },
+      },
+      (res) => {
+        res.on('data', (chunk) => {
+          console.log(chunk);
+        });
+      }
+    );
+    req.on('error', (e) => {
+      console.log(e);
+    });
+    req.write(body);
+
+    req.end();
+  }
+});
+```
+
+You will see the following response body on a successfull request:
+
+```json
+{
+  "id": "a438bfd2-ff4c-4c82-bcec-c18115e6779b",
+  "entity": {
+    "kind": "Resource",
+    "spec": { "type": "aws-account", "owner": "dx-team" },
+    "metadata": {
+      "name": "Development Account",
+      "annotations": {
+        "aws-account/arn": "arn:aws:organizations::111111111111:account/o-exampleorgid/333333333333",
+        "backstage.io/managed-by-location": "roadie-api:/api/catalog/roadie-entities/entities/by-ref/resource%3Adefault%2Fdevelopment%20account",
+        "backstage.io/managed-by-origin-location": "roadie-api:/api/catalog/roadie-entities/entities"
+      },
+      "description": "AWS accounts"
+    },
+    "apiVersion": "backstage.io/v1alpha1"
+  },
+  "entityRef": "resource:default/development account",
+  "rawData": "resource:default/development account",
+  "set": null,
+  "updatedBy": "user:default/bob",
+  "source": "api-entity",
+  "updatedAt": "2024-02-15T13:31:20.718+00:00"
+}
+```
+
+Go to your catalog page (`Catalog` -> `Resource`) and you will immedietly see these `Resource` entities in your catalog.
+![aws-account Resource entities](aws-accounts-in-catalog.png)
+
+## Keep it syncing
+
+To continuesly update your catalog with the changes in your AWS accounts you will need to run this script on a schedule. I advise you to use your organization's best practice to run these scheduled jobs.
+
+Regardless how it will run you will need to be sure that environment has the correct tokens to be able to fetch from your organization's AWS and to be able to push to your Roadie api. Do not forget to configure your `ROADIE_API_TOKEN` and your `AWS_PROFILE`
```

Comment:
```suggestion
Regardless how it will run you will need to be sure that environment has the correct tokens to be able to fetch from your organization's AWS and to be able to push to your Roadie API. Do not forget to configure your `ROADIE_API_TOKEN` and your `AWS_PROFILE`
```